internal/store/store.go

```diff
@@ -297,6 +306,26 @@ response:
 	return nil
 }
 
+/**
+* Creates client for v2 generated API.
+ */
+func (s *Store) createV2Client(client *http.Client) error {
+	modifiers := []apiv2.ClientModifier{
+		apiv2.UseHTTPClient(client),
+		apiv2.UseUserAgent(config.UserAgent),
+		apiv2.UseDebug(log.IsDebugLevel())}
```

Comment:
[q] Shouldn't we set versioning as well?

Reply:
What do you mean by versioning? 
If you mean sending information about version to backend?

The client does that at the moment - each client does include a version of the versioned API.
We have also follow up project for some versioning features for atlas cli.

Reply:
I meant setting the `application/vnd.atlas.2022-10-15+json` in the header of the request.

> The client does that at the moment - each client does include a version of the versioned API.

Ah okay. Thanks for the clarifications.